firmware/quadruna/VC/src/io/io_imu.c

```diff
@@ -31,7 +34,7 @@ bool io_imu_getLinearAccelerationX(float *x_acceleration)
 
     // Convert raw value to acceleration in m/s^2
     int16_t x_raw   = (int16_t)(x_data[1] << 8 | x_data[0]);
-    *x_acceleration = x_raw * SENSITIVITY * 9.81f / 1000.0f;
+    *x_acceleration = x_raw * ACCEL_SENSITIVITY * 9.81f / 1000.0f;
```

Comment:
Change based on requested changes, also make sure u build and run tests to catch issues